Documentation/release-notes/4567.md

```diff
@@ -0,0 +1,18 @@
+### Preliminary JDK 11 Support
+
+Xamarin.Android can now use JDK 11 to build libraries and applications.
+
+To do so:
+
+ 1. Install JDK 11
+ 2. Set the `$(JavaSdkDirectory)` MSBuild property to refer to the root
+    of the JDK installation from (1).
```

Comment:
Can project authors configure the JDK via the Java Development Kit location setting in Visual Studio options (for example, on Windows: **Tools > Options > Xamarin > Android Settings > Java Development Kit Location**)? If so, I think it could be good to mention that approach first.

Then, if a different approach might work best on CI systems, such as Azure Pipelines, that could be included as a second option.

Reply:
Looks like these locations have documentation for how to set that value:

  * https://docs.microsoft.com/en-us/xamarin/android/troubleshooting/questions/update-jdk?tabs=windows
  * https://docs.microsoft.com/en-us/xamarin/android/get-started/installation/windows#configuration